test/data_processing/test_nodes.py

```diff
@@ -460,3 +461,17 @@ def test_restless_process_2(self):
         # classification: ["11", "00", "10", "00", "11", "00", "10", "00"]
         expected_data = np.array([{"10": 2, "11": 2}, {"00": 4}])
         self.assertTrue(processed_data.all() == expected_data.all())
+
+    def test_restless_iq_process(self):
```

Comment:
Can you add a test for 2 qubits?

Reply:
Yes, added one test here: [b768968](https://github.com/Qiskit/qiskit-experiments/pull/752/commits/b76896896dc1d1d5e0c918f5ad6baa9fbcebc2b8)